guides/issuing-assets.md

```diff
@@ -252,26 +252,25 @@ server.submitTransaction(setHomeDomain);
-
-### Specialized Issuing Accounts
-
-In the simplest situations, you can issue assets from your everyday Stellar account. However, if you operate a financial institution or a business, you should keep a separate account specifically for issuing assets. Why?
-
-- Easier tracking: because an asset represents a credit, it disappears when it is sent back to the account that issued it. To better track and control the amount of your asset in circulation, pay a fixed amount of the asset from the issuing account to the working account that you use for normal transactions.
-
-  The issuing account can issue the asset when more of the underlying value (like actual bananas or dollar bills) is on hand and the accounts involved in public transactions never have to worry about how much is available outside Stellar.
 
-- Keeping trust simple: as your usage of Stellar grows, you might consider having multiple accounts for a variety of reasons, such as making transactions at high rates. Keeping a canonical issuing account makes it easier for others to know which account to trust.
+## Requiring or Revoking Authorization
 
+Accounts have [several flags](concepts/accounts.md#flags) related to issuing assets.
 
-### Requiring or Revoking Authorization
+If you’d like to control who can be paid with your asset, or if your asset has some special purpose
+requiring sign-off from you, use the
+[`AUTHORIZATION REQUIRED` flag](concepts/assets.md#controlling-asset-holders), which requires that
+the issuing account also approves a trustline before the receiving account is allowed to be paid
+with the asset.
 
-Accounts have [several flags](concepts/accounts.md#flags) related to issuing assets. Setting the [`AUTHORIZATION REVOCABLE` flag](concepts/assets.md#revoking-access) allows you to freeze assets you issued in case of theft or other special circumstances. This can be useful for national currencies, but is not always applicable to other kinds of assets.
+[`AUTHORIZATION REVOCABLE` flag](concepts/assets.md#controlling-asset-holders) allows you to freeze
+assets you issued in case of theft or other special circumstances. This can be useful for national
+currencies, but is not always applicable to other kinds of assets.
 
-If your asset is special purpose or you’d like to control who can be paid with it, use the [`AUTHORIZATION REQUIRED` flag](concepts/assets.md#controlling-asset-holders), which requires that the issuing account also approves a trustline before the receiving account is allowed to be paid with the asset.
+For most cases, you should avoid setting `AUTHORIZATION REVOCABLE` on your asset. In the past, some
+issuers have used the `AUTHORIZATION REVOCABLE` flag in order to impose lock-up periods. However,
+this is a problematic mechanism because it does not provide the user any guarantees with regard to
+when or if the assets will be unlocked.
```

Comment:
There's another reason why it's problematic. It requires serious plumbing in order to undo.
Just turning the `AUTH_REQUIRED`/`AUTH_REVOCABLE` flags to `false` doesn't affect existing accounts. You'll need to manually `ALLOW_AUTH=true` on existing accounts which can only be done when the `AUTH_REQUIRED` flag is true. Which essentially means that, to do it correctly without affecting new accounts being created, you'll need to execute transactions that sandwich `ALLOW_AUTH=true` operations between setting `AUTH_REQUIRED=true` in the beginning and changing to `AUTH_REQUIRED=false` in the end.